docs/Other SDL Features/OEM Exclusive Apps/index.md

```diff
@@ -0,0 +1,74 @@
+# OEM Exclusive Apps
+The Android library can prevent apps from starting their `SdlRouterService` based on if they support the vehicle type if the connection is over Bluetooth.
+
+!!! NOTE
+If older apps are installed that use a version less then SDL Android 4.12.0, on the first connection to the vehicle an OEM app may host the `SdlRouterService` but on subsequent connections it may pass it off to another app.
+!!!
+
+!!! NOTE
+This does not work over AOA connections.
+!!!
+
+
+## Create file for supported vehicle types and add to Android Manifest
+To implement this feature, you will need to define an XML file for supported vehicles in resources of the project called `supported_vehicle_type.xml`, and add it as metaData for `SdlRouterService` in its AndroidManifest.  If an app defines a vehicle-type element, then it should always have a make attribute, all other attributes are optional. However, if the app developers want to use model year or trim, they should define make and model attributes as well. The Java Suite app library will check only the defined attributes. The below example shows a valid vehicle type resource file.
```

Comment:
```suggestion
To implement this feature, you will need to define an XML file for supported vehicles in resources of the project called `supported_vehicle_type.xml`, and add it as metaData for `SdlRouterService` in its AndroidManifest.  If an app defines a vehicle-type element, then it should always have a make attribute; all other attributes are optional. However, if you want to use model year or trim, you should define make and model attributes as well. The Java Suite app library will check only the defined attributes. The below example shows a valid vehicle type resource file.
```
I think this change makes sense because the guide is being written as instructions for app developers.